### PR TITLE
[GUI] new feature to search for parameter groups

### PR DESCRIPTION
**Pull request by donovaly**
see https://forum.freecadweb.org/viewtopic.php?f=17&t=44169

### CODE REVIEW

File: src/Gui/DlgParameterImp.cpp [hunk 1]
```diff
@@ -117,6 +117,21 @@ DlgParameterImp::DlgParameterImp( QWidget* parent,  Qt::WindowFlags fl )
     connect(paramGroup, SIGNAL(currentItemChanged(QTreeWidgetItem*, QTreeWidgetItem*)), 
             this, SLOT(onGroupSelected(QTreeWidgetItem*)));
     onGroupSelected(paramGroup->currentItem());
+
+    connect(ui->findGroupLE, SIGNAL(textChanged(QString)),
+        this, SLOT(on_findGroup_changed(QString)));
+
+    // setup for function on_findGroup_changed:
+    // store the current font properties because
+    // we don't know what style sheet the user uses for FC
```
Comment (luzpaz) [inline src/Gui/DlgParameterImp.cpp]: `s/style sheet/stylesheet/`

Comment (donovaly) [reply src/Gui/DlgParameterImp.cpp]: What do you mean?, "style sheet" is the right term: https://en.wikipedia.org/wiki/Style_sheet

Comment (luzpaz) [reply src/Gui/DlgParameterImp.cpp]: @donovaly I guess both are used interchangeably in the FC code. Sorry for the noise.

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**